README.md

```diff
@@ -64,32 +64,28 @@ Please read the Requirements section. Thank you!
 7. Select the volume containing the bootable installer. Then click the up arrow or press Return. 
 8. Choose your language, if prompted.
 9. Open the Terminal.
-10. Execute `cd /Volumes/bigmac; .preinstall.sh`, and quit Terminal.
+10. Execute `/Volumes/bigmac/preinstall.sh`, and quit Terminal.
 11. Select Install macOS Big Sur from the Utilities window, then click Continue and follow the onscreen instructions.
 
-Legacy USB 2.0 for keyboard and mouse devices is not working 100%. If you leave them plugged into a USB 2.0 port, they will work. If you disconnect them, they won't reconnect. This is a problem with Big Sur which Apple introduced on their own. A workaround is use a USB3.0 PCIe card but this only works when the system is booted, not from a cold start. USB 3.0 hubs plugged into a USB 2.0 port may also work. A patch may be implemented to take care of this issue.
+Legacy USB 2.0 for keyboard and mouse devices is never working. If you leave them plugged into a USB 2.0 port, they will work. If you disconnect them, they won't reconnect. This is a problem with Big Sur which Apple introduced on their own. A workaround is use a USB3.0 PCIe card but this only works when the system is booted, not from a cold start. USB 3.0 hubs plugged into a USB 2.0 port may also work. A patch may be implemented to take care of this issue.
```

Comment:
please do not use never. It is too strong of a word.

Reply:
```suggestion
Legacy USB 2.0 for keyboard and mouse devices is not working. If you leave them plugged into a USB 2.0 port, they will work. If you disconnect them, they won't reconnect. This is a problem with Big Sur which Apple introduced on their own. A workaround is use a USB3.0 PCIe card but this only works when the system is booted, not from a cold start. USB 3.0 hubs plugged into a USB 2.0 port may also work. A patch may be implemented to take care of this issue.
```